docs/shared/ccloud_setup.md

```diff
@@ -1,3 +1,3 @@
-Setup your environment in [Confluent Cloud](https://www.confluent.io/confluent-cloud/tryfree/), a fully-managed Apache Kafka service.
+Setup your environment in [Confluent Cloud](https://www.confluent.io/confluent-cloud/tryfree/), a fully-managed Apache Kafka® service. Within your environment create a [ksqlDB](https://ksqldb.io/) application.
```

Comment:
```suggestion
Setup your environment in [Confluent Cloud](https://www.confluent.io/confluent-cloud/tryfree/), a fully-managed Apache Kafka® service, and within your environment provision a [ksqlDB](https://ksqldb.io/) application.
```

Reply:
I like your edit, but WDYT about sticking w/ the term "Create" since that's what's used in the ksqlDB cloud product?

Reply:
Ok